test/library/standard/Math/promotion/generate_tests/make_promotion_tests.py

```diff
@@ -31,6 +29,8 @@
             ('log10', 1.0, 1000.0),
             ('log1p', 1.0, 10.0),
             ('log2', 1.0, 1024.0),
+            ('nearbyint', -7.0, 7.0),
+            ('rint', -7.0, 7.0),
```

Comment:
Should divceil* and divfloor* be added here? Or not needed

Reply:
The README in the test directory for generating the script says it's only intended to handle monadic functions, where divceil* and divfloor* both take two arguments.  So it would need to live in a separate place and be handled by an additional script.  Would you be okay with me adding that as one of our 2.0 validation steps rather than handling it now?

Reply:
Yes that's ok with me